versions.json

```diff
@@ -1,6 +1,6 @@
 {
   "abigen": "v1.10.25",
-  "foundry": "84d98427e17370ff08ec34f00e3c7e539753a760",
+  "foundry": "0.2.0 (a5efe4f8f425e2f6fb35b0e298f0f46acce11dad 2024-02-08T00:16:00.244105000Z)",
```

Comment:
Can we leave this as the commit hash? its going to be annoying ux to update this. will need to download and run version command to get the output instead of just copy/paste from github. this doesn't need to care about the time, whether its newer or older, it just needs to check if it doesn't match the commit sha

Reply:
Yea this actually needs to be _only_ the commit hash because:
- For now the all foundry releases are `0.2.0` because forge uses nightly releases
- The date/time is when you built the binary. So it will differ for each OS's release, and it will differ if you built locally vs. pulled a binary from github releases

Reply:
Ok, updated